.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 .idea
+.class
```

Comment:
I'm not familiar with Maven, but normally any files produced as part of a build process should end up in their own directory (e.g. `/build), in which case that entire directory could be added to .gitignore.

Where exactly do the `.class` files end up when you build?

Reply:
I've been doing some testing with individual java files where I've been compiling and running them, and the .class files end up in the same directory as the the original java file. So far I've just been watching to make sure I don't commit those files, but I wanted to add this into the .gitignore to prevent them from accidentally getting uploaded to the repo.

Reply:
Got it, thanks!